test/integration/worldwide_organisation_test.rb

```diff
@@ -52,6 +52,7 @@ class WorldwideOrganisationTest < ActionDispatch::IntegrationTest
 
   test "renders the navigational corporate information pages" do
     setup_and_visit_content_item("worldwide_organisation")
+    puts page.body
```

Comment:
Debug?

Reply:
Oh sorry, this is still in draft, apologies. Change looks good though 👍

Reply:
Ready now! Sorry I was a bit too keen with making it a PR 😅